docs/Buzzer.md

```diff
@@ -44,6 +44,38 @@ Sequences available in Cleanflight v1.9 and above are :
     15   ARMED                 0, 245, 10, 5	Board is armed (after pause ; repeats until board is disarmed or throttle is increased)
 
 
+### CLI " beeper "  command
```

Comment:
beeper vs. buzzer, I´d prefer if the same terminoliogy was used. Now there is two words for the same thing.